Assets/Mirror/Runtime/NetworkClient.cs

```diff
@@ -36,13 +37,19 @@ internal void SetHandlers(NetworkConnection conn)
         public NetworkClient()
         {
             if (LogFilter.Debug) { Debug.Log("Client created version " + Version.Current); }
-            AddClient(this);
+
+            if (singleton != null)
+            {
+                Debug.LogError("NetworkClient: can only create one!");
+                return;
+            }
+            singleton = this;
         }
 
-        public NetworkClient(NetworkConnection conn)
+        // initialize with connection. calls NetworkClient() to setup singleton.
+        public NetworkClient(NetworkConnection conn) : this()
         {
             if (LogFilter.Debug) { Debug.Log("Client created version " + Version.Current); }
```

Comment:
remove this line,   the other constructor already logs this

Reply:
done